client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx

```diff
@@ -40,9 +40,7 @@ const Paid: React.FC< OwnProps > = ( {
 			: 1 - INTRO_PRICING_DISCOUNT_PERCENTAGE / 100;
 
 	const couponOriginalPrice = parseFloat( ( discountedPrice ?? originalPrice ).toFixed( 2 ) );
-	const couponDiscountedPrice = parseFloat(
```

Comment:
This inexplicably rounded 49.975 up, will prices such as 4.975 were round down.

Reply:
🙃